test/suites/test_schema.py

```diff
@@ -43,6 +44,8 @@ def setUpClass(self):
         self.srv.start()
         self.con = tarantool.Connection(self.srv.host, self.srv.args['primary'],
                                         encoding=self.encoding)
+        self.con_schema_disable = tarantool.Connection(self.srv.host, self.srv.args['primary'],
+                                                       encoding=self.encoding, fetch_schema=False)
```

Comment:
`authenticate` reloads schema. For now, it will fail on attempt to subscribe `self.schema` in `load_schema`. So your tests have two major issues (related to each other):
- There is no authenticate. `self.con` authenticates in `test_00` -- it is a very weird and misleading approach to use separate test case to prepare something and use `test_N` naming to rely on their tests. I think we may simply add `user`/`password` here.
- There is no check that every other handle is ok. For example, because of missing authenticate, `eval` throws `tarantool.error.DatabaseError: (42, "Execute access to universe '' is denied for user 'guest'")`.

We must fix both of them.

Reply:
Let's also test it        like crud: for all three types of connection
https://github.com/tarantool/tarantool-python/blob/6f0d5369ab115c8e7d456e8b43ae77d80f5de705/test/suites/test_crud.py#L711

Reply:
Done
* Test case `test_00` has been deleted. Authentication is performed when opening connections to test instances
* Now not only the unavailability of some methods is checked, but also the availability of those that should not have been affected
* Implemented with `connection`, `mesh_connection` and `pool_connection`